.github/workflows/cuda12.1-whl-release.yml

```diff
@@ -74,11 +74,16 @@ jobs:
           use-github-cache: false
       - name: Build wheel
         run: |
+          ((Get-Content -path CMakeLists.txt -Raw) -replace '-Wall','/W0') | Set-Content CMakeLists.txt
           mkdir build
           cd build
           pip install -U setuptools
           ..\builder\windows\generate.ps1
-          cmake --build . --config Release -- /m > build.log.txt
+          cmake --build . --config Release -- /m /v:q
```

Comment:
could remove this line so as to avoid manually upload files to release page when the manually start this workflow
https://github.com/InternLM/lmdeploy/blob/2831dc241063b3e26e1baf581a039a387d9f333e/.github/workflows/cuda12.1-whl-release.yml#L106

Reply:
When manually start this workflow, we can select specific tag. I don't think use a custom tag to this workflow is a good idea since the build files are not correspond to their actual commits.

Reply:
Ok.  Considering this case is less normal, so manually downloadings and uploadings wheel files is acceptable. BTW,  just FYI,  a `tag_name` could be passed to this step per [here](https://github.com/softprops/action-gh-release?tab=readme-ov-file#inputs).